theme/common.less

```diff
@@ -27,13 +27,7 @@ html, body {
 }
 
 #filelist, #maineditor, #sidedocs {
-    top:@mainMenuHeight;
-}
-
-.hideMenuBar #filelist,
```

Comment:
Keep these up here for the .hideMenuBar feature. 
Add top:0 to the notificationBanner CSS. 

we have CSS optimizers in place so it shouldn't matter.